metadata.tvshows.themoviedb.org.python/resources/settings.xml

```diff
@@ -146,6 +146,25 @@
 					<default>true</default>
 					<control type="toggle"/>
 				</setting>
+        <setting id="enab_trailer" type="boolean" label="30305" help="">
```

Comment:
Same whitespace mixing going on

Reply:
I'll fix that for the next build.  I accepted a PR for the trailer stuff, and my bet is they used spaces in the XML when I always (or just about always) use tabs in XML.